samples/word/50-document/insert-external-document.yaml

```
@@ -71,7 +89,12 @@ template:
                 <br>
                 <p>Insert the body text from the selected document.</p>
                 <button id="insert-document" class="ms-Button">
-            <span class="ms-Button-label">Insert document</span>
+          <span class="ms-Button-label">Insert document (no settings)</span>
+          </button>
+                <br>
+                <p>Insert the body text with chosen settings applied from the selected document.</p>
```

Since the user isn't "choosing" the settings, I'd omit that word.
```suggestion
                <p>Insert the body text with settings applied from the selected document.</p>
```